File: app/src/main/java/com/example/petcare/MainActivity.java
package com.example.petcare;

import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.os.Handler;
import android.view.WindowManager;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    private SharedPreferences sharedPreferences;
    private Boolean firstTime;

    private static final int SPLASH_SCREEN_TIMEOUT = 2000;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        getWindow().setFlags(
                WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN
        );
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        sharedPreferences = getSharedPreferences("MyPrefs", MODE_PRIVATE);
        firstTime = sharedPreferences.getBoolean("firstTime", true);

        if (firstTime) {
            new Handler().postDelayed(new Runnable() {
                @Override
                public void run() {
                    SharedPreferences.Editor editor = sharedPreferences.edit();
                    firstTime = false;
                    editor.putBoolean("firstTime", firstTime);
                    editor.apply();

                    Intent mySuperIntent = new Intent(MainActivity.this, AppIntro.class);
                    startActivity(mySuperIntent);
                    showInterstitial();
                    finish();
                }
            }, SPLASH_SCREEN_TIMEOUT);
        } else {
            Intent intent = new Intent(MainActivity.this, MainActivity2.class);
            overridePendingTransition(0, 0);
            startActivity(intent);
            showInterstitial();
            finish();
        }
    }

    private void showInterstitial() {

    }
}

File: app/src/main/java/com/example/petcare/skeniranje/MainActivity.kt
package com.example.petcare.skeniranje

import android.os.Bundle
import android.view.WindowManager
import androidx.appcompat.app.AppCompatActivity
import com.example.petcare.databinding.ActivityMainScanPlantBinding

@Suppress("DEPRECATION")
class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainScanPlantBinding

    override fun onCreate(savedInstanceState: Bundle?) {

        window.setFlags(

            WindowManager.LayoutParams.FLAG_FULLSCREEN,

            WindowManager.LayoutParams.FLAG_FULLSCREEN

        )

        super.onCreate(savedInstanceState)

        binding = ActivityMainScanPlantBinding.inflate(layoutInflater)

        val view =  binding.root

        setContentView(view)

    }

}
